src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -214,7 +214,7 @@ def _convert_non_diffusers_lora_to_diffusers(state_dict, unet_name="unet", text_
     if len(state_dict) > 0:
         raise ValueError(f"The following keys have not been correctly renamed: \n\n {', '.join(state_dict.keys())}")
 
-    logger.info("Kohya-style checkpoint detected.")
+    logger.info("Non-diffusers checkpoint detected.")
```

Comment:
At this point a checkpoint can be of any supported type: [BenTrainer](https://github.com/TheLastBen/fast-stable-diffusion), Kohya, OneTrainer, etc.